### PR TITLE
Style updates bubble choice

### PR DESCRIPTION
**Pull request by dmcavoy**
A couple style improvements for the cards that show up on Lesson Extras and Bubble Choice levels.

- Made the sublevel cards a little bigger (Made the lesson extras card as big as can fit in the space)
- Made the thumbnail image bigger
- Adding some more spacing between lines of a long title
- Fixed the spacing to the right of the Flag Icon for Lesson Extras

Bubble Choice Example with max length display name and description: 
| Before | After |
| --- | --- |
| Screen Shot 2020-05-11 at 4 00 12 PM |  Screen Shot 2020-05-11 at 3 55 18 PM |

Lesson Extras Example with max length display name and description: 
| Before | After |
| --- | --- |
| Screen Shot 2020-05-11 at 4 00 27 PM |  Screen Shot 2020-05-11 at 3 51 50 PM |

# Lesson Extras and Bubble Choice Editing Updates

- Updated text on the level edit page to make the controls clearer
- Put character limits on the display name and the description (will not impact existing names or descriptions until you edit them).

Screen Shot 2020-05-11 at 4 02 50 PM

## Links

- [jira](https://codedotorg.atlassian.net/browse/LP-1443)

## Testing story

Mostly UI changes and levelbuilder areas that I couldn't test. 

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: dashboard/db/schema.rb [hunk 1]
```diff
@@ -1755,6 +1755,14 @@
...
   end
 
+  add_foreign_key "ap_school_codes", "schools"
+  add_foreign_key "census_inaccuracy_investigations", "census_overrides"
+  add_foreign_key "census_inaccuracy_investigations", "census_submissions"
+  add_foreign_key "census_inaccuracy_investigations", "users"
+  add_foreign_key "census_overrides", "schools"
+  add_foreign_key "census_submission_form_maps", "census_submissions"
+  add_foreign_key "census_summaries", "schools"
+  add_foreign_key "circuit_playground_discount_applications", "schools"
```
Comment (clareconstantine) [inline dashboard/db/schema.rb]: did you mean to include these?

File: apps/src/code-studio/components/SublevelCard.jsx [hunk 2]
```diff
@@ -50,7 +51,6 @@ const styles = {
     width: 30
   },
   column: {
-    width: 280,
     marginLeft: MARGIN * 2,
```
Comment (clareconstantine) [inline apps/src/code-studio/components/SublevelCard.jsx:54]: should the `margin: 10` on line 58 be `margin: MARGIN`?

File: apps/src/code-studio/components/SublevelCard.jsx [hunk 3]
```diff
@@ -8,17 +8,18 @@ import LessonExtrasFlagIcon from '@cdo/apps/templates/progress/LessonExtrasFlagI
...
 const styles = {
   row: {
     display: 'flex',
+    width: WIDTH,
     marginBottom: MARGIN,
     backgroundColor: color.white,
     border: '1px solid rgb(187, 187, 187)',
     borderRadius: 2,
-    width: 400,
     marginRight: 10
```
Comment (clareconstantine) [inline apps/src/code-studio/components/SublevelCard.jsx]: should this be `margin: MARGIN`? How did you decide which places to use the const and which places to leave it as 10?

**Review by clareconstantine — COMMENTED** [review]
This looks good, just two small questions and some extra schema.rb changes

File: dashboard/app/views/levels/editors/fields/_bubble_choice_sublevel.html.haml [hunk 4]
```diff
@@ -1,18 +1,18 @@
...
+  Bubble Choice & Lesson Extras
 
 #bubble_choice.in.collapse
   - unless @level.is_a?(DSLDefined) || @level.is_a?(Unplugged)
     .field
       = f.label :display_name
-      %p Set the name that will show for this level on the parent bubble choice level. Also will override how this level is displayed on the Course Overview page.
-      = f.text_field :display_name, placeholder: '(defaults to Name)'
+      %p Set the name that will show for this level on the parent level. Also will override how this level is displayed on the Course Overview page. Limit to 25 characters
+      = f.text_field :display_name, placeholder: '(defaults to Name)', maxlength: 25, style: 'width: 50%'
```
Comment (Erin007) [inline dashboard/app/views/levels/editors/fields/_bubble_choice_sublevel.html.haml:9]: While this ever be translated? I'm just wondering what happens if something 25 characters long in English is longer in a different language. Same for description below.

**Review by Erin007 — APPROVED** [review]
question about string size limits re: translatability, but LGTM!

File: apps/src/code-studio/components/SublevelCard.jsx (hunk 3 above)
Comment (dmcavoy) [reply apps/src/code-studio/components/SublevelCard.jsx]: I think only the left and right margins are important to be using MARGIN but thats how we calculate some sizing stuff to make sure the widths of everything is right. I'll double check all the places that should be using it are.

File: dashboard/app/views/levels/editors/fields/_bubble_choice_sublevel.html.haml (hunk 4 above)
Comment (dmcavoy) [reply dashboard/app/views/levels/editors/fields/_bubble_choice_sublevel.html.haml:9]: We don't currently support editing in any other language and we are a good ways off from that. 

I talked to Amanda about this and she said at the place she worked before they used a general rule that translations were generally max 20% longer than the english version so she suggested that I find out the max length of english characters that could fit in the space and then multiple it by 0.8 to find the max length we would enforce in order to leave extra space for translation.

File: dashboard/db/schema.rb (hunk 1 above)
Comment (dmcavoy) [reply dashboard/db/schema.rb]: Sorry about that. My local environment was a mess and I think I forgot to merge staging in to this branch after fixing that up. Should be all set now.

File: dashboard/app/views/levels/editors/fields/_bubble_choice_sublevel.html.haml (hunk 4 above)
Comment (Erin007) [reply dashboard/app/views/levels/editors/fields/_bubble_choice_sublevel.html.haml:9]: That's a nifty rule of thumb that might come in handy other places! Thanks for following up.

File: apps/src/code-studio/components/SublevelCard.jsx (hunk 2 above)
Comment (dmcavoy) [reply apps/src/code-studio/components/SublevelCard.jsx:54]: Updated

File: apps/src/code-studio/components/SublevelCard.jsx (hunk 3 above)
Comment (dmcavoy) [reply apps/src/code-studio/components/SublevelCard.jsx]: Updated

**Pull request merged**